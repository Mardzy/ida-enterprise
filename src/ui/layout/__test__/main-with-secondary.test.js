import MainWithSecondary from "../main-with-secondary";

describe("The main with secondary content", () => {

  it("renders both sections", () => {
    const main = () => <div className="main">Main</div>;
    const secondary = () => <div className="secondary">Secondary</div>;

    const wrapper = mount(
      <MainWithSecondary main={main} secondary={secondary}/>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});