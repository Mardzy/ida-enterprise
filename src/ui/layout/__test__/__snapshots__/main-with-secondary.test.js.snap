// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The main with secondary content renders both sections 1`] = `
<MainWithSecondary
  main={[Function]}
  secondary={[Function]}
>
  <div
    className="main"
  >
    Main
  </div>
  <div
    className="secondary"
  >
    Secondary
  </div>
</MainWithSecondary>
`;
